conversions/fahrenheit_to_celsius.py

```diff
@@ -0,0 +1,39 @@
+""" Convert temperature from Fahrenheit to Celsius """
+
+
+def fahrenheit_to_celsius(fahrenheit):
+    """
+    Convert a given value from Fahrenheit to Celsius and round it to 2 d.p.
+
+    >>> fahrenheit_to_celsius(0)
+    -17.78
+    >>> fahrenheit_to_celsius(20)
+    -6.67
+    >>> fahrenheit_to_celsius(40)
+    4.44
+    >>> fahrenheit_to_celsius(60)
+    15.56
+    >>> fahrenheit_to_celsius(80)
+    26.67
+    >>> fahrenheit_to_celsius(100)
+    37.78
+    >>> fahrenheit_to_celsius("100")
+    Traceback (most recent call last):
+    ...
+    TypeError: 'str' object cannot be interpreted as integer
+    """
+    if type(fahrenheit) == str:
+        """
+        Check whether given value is string and raise Type Error
+        """
+        raise TypeError("'str' object cannot be interpreted as integer")
+
+    celsius = (fahrenheit - 32) * 5 / 9  # value converted from fahrenheit to celsius
+    celsius = round(celsius, 2)
+    # converted (celsius) value is rounded to two decimal places
+    print(celsius)
```

Comment:
According to CONTRIBUTING.md, algorithm functions should not `print()`.

Instead, please return a float and let the caller do the printing.  Like `print(fahrenheit_to_celsius(96))`

https://en.wikipedia.org/wiki/96%C2%B0_in_the_Shade